gobblin-modules/gobblin-metadata/src/main/java/gobblin/metadata/types/GlobalMetadata.java

```diff
@@ -222,7 +222,23 @@ public void setContentType(String contentType) {
    * @return
    */
   public String getContentType() {
-    return (String)getDatasetMetadata(CONTENT_TYPE_KEY);
+    return (String) getDatasetMetadata(CONTENT_TYPE_KEY);
+  }
+
+  public void setPermission(String permission) {
+    setDatasetMetadata(PERMISSION_KEY, permission);
```

Comment:
Can we talk about how ACLs are expressed? @eogren @htran1 might also be interested in this.